assets/dev/js/editor/controls/color.js

```diff
@@ -162,22 +162,18 @@ export default class extends ControlBaseDataView {
 	}
 
 	// Create the markup for the colors in the global select dropdown.
-	buildGlobalsList( globalColors ) {
-		const $globalColorsPreviewContainer = jQuery( '<div>', { class: 'e-global__preview-items-container' } );
-
+	buildGlobalsList( globalColors, $globalPreviewItemsContainer ) {
 		Object.values( globalColors ).forEach( ( color ) => {
 			if ( ! color.value ) {
 				return;
 			}
 
 			const $color = this.createGlobalItemMarkup( color );
 
-			$globalColorsPreviewContainer.append( $color );
+			$globalPreviewItemsContainer.append( $color );
 		} );
 
-		this.ui.$globalColorsPreviewContainer = $globalColorsPreviewContainer;
-
-		return $globalColorsPreviewContainer;
+		return $globalPreviewItemsContainer;
```

Comment:
Now you don't have to return it

Reply:
I do. The return result gets appended in `addGlobalsListToPopover()`